package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "niftyswap",
-  "version": "1.0.1",
+  "version": "1.0.1-a",
```

Comment:
Maybe double check that the bytecode doesn't change, because if it did we should do a major version bump.